src/components/tile/tile.pw.tsx

```diff
@@ -277,6 +277,28 @@ test.describe("Tile component", () => {
     });
   });
 
+  borderVariants
```

Comment:
**suggestion**: given that we are validating the styling when the `borderVariant`and `variant` props are set (in tests on lines 90 and 266), I'm thinking about the additional value brought in by this parameterised test, apart from ensuring the previous override behaviour doesn't reoccur.

As an alternative, what do you think about modifying the test on line 90 to remove checking the border colour? This way, we could avoid redundant verification of the border colour.